srml/support/src/traits.rs

```diff
@@ -135,28 +136,36 @@ pub trait Currency<AccountId> {
 	///
 	/// If `who` doesn't exist, it is created
 	///
-	/// Returns if the account was successfully updated or update has led to killing of the account.
+	/// # NOTES
 	///
-	/// NOTE: This assumes that the total stake remains unchanged after this operation.
+	/// This assumes that the total stake remains unchanged after this operation.
```

Comment:
total stake -> total issuance
What is `this operation`? Sum of all balances will be changed after this method and it will mismatch to `TotalIssuance`. The invariant here is total issuance should be consistent (match to sum of all balances) this transaction is processed.

Reply:
yes it seems it shouldn't exist but I'm not sure: https://github.com/paritytech/substrate/issues/1979

Reply:
Resolved with https://github.com/paritytech/substrate/pull/1943/commits/5e28748b8ba582b40f9a271c340e8f7f8aa0c0b8